governance/default_test.go

```diff
@@ -119,6 +119,12 @@ var tstData = []voteValue{
 	{k: "reward.ratio", v: "0/0/0", e: false},
 	{k: "reward.ratio", v: "30.5/40/29.5", e: false},
 	{k: "reward.ratio", v: "30.5/40/30.5", e: false},
+	{k: "reward.kip82ratio", v: "20/80", e: true},
```

Comment:
Should we add ratio to 100/0 as default changed?

Reply:
It's reverted back to 20/80 in 0b3e45e2

Reply:
You can check https://github.com/klaytn/klaytn/pull/1631#discussion_r1024772248